subprojects/docs/src/docs/release/release-features.txt

```diff
@@ -0,0 +1,4 @@
+ - Incremental Java compilation by default
+ - Automatic Gradle caches/ cleanup
```

Comment:
`- Periodic cache cleanup`?

Reply:
Perhaps "periodic" is better, but I'm also hoping to make it a little more specific by saying "Gradle" and "caches/" dir (knowing full well it's caches\ on Windows)

Reply:
"Periodic Gradle caches cleanup"?

Reply:
It's not just the `caches` dir, it's also `wrapper/dists`. 😉 

I'll leave it up to you but I found the trailing slash confusing. Looked like a typo to me.